docs/developer_guide.md

```diff
@@ -1,13 +1,13 @@
 # Developer guide
 
 ## Prerequisites
-- docker
+- Docker
 - opm https://github.com/operator-framework/operator-registry/releases
 - operator-sdk https://github.com/operator-framework/operator-sdk/releases
+- olm https://github.com/operator-framework/operator-lifecycle-manager/releases
 - golang 1.19
 - make
-- kubectl
```

Comment:
So it's not needed in any of mentioned make targets?

Reply:
My mistake, it is, I'll bring it back.